server/src/main/java/org/apache/iotdb/db/qp/physical/sys/TracingPlan.java

```diff
@@ -38,6 +38,11 @@ public TracingPlan(boolean isTracingOn) {
     return new ArrayList<>();
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return new ArrayList<>();
```

Comment:
And here.

Reply:
I'll change it. Thanks for your suggestion.

Reply:
I'll change it. Thanks for your suggestion.